### PR TITLE
Generate the MySQL SDKs 0.1.0 public preview

### PR DESCRIPTION
**Pull request by gechris**
Generate the MySQL SDKs 0.1.0 public preview

### CODE REVIEW

File: sdk/mysql/Microsoft.Azure.Management.MySQL/src/Microsoft.Azure.Management.MySQL.csproj [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+  <PropertyGroup>
+    <PackageId>Microsoft.Azure.Management.MySQL</PackageId>
+    <Description>Provides developers with libraries for the Azure Database for MySQL under Azure Resource manager to deploy an instance of MySQL. Create, Update, Delete, Restore, and more.</Description>
+    <Version>0.1.0-preview</Version>
+    <AssemblyName>Microsoft.Azure.Management.MySQL</AssemblyName>
+    <PackageTags>MySQL;Azure Database for MySQL;</PackageTags>
+    <PackageReleaseNotes>
+      <![CDATA[
+        This is a release of the Azure Database for MySQL SDK.
```
Comment (allenjzhang) [inline sdk/mysql/Microsoft.Azure.Management.MySQL/src/Microsoft.Azure.Management.MySQL.csproj:14]: More detailed release notes?

File: sdk/mysql/Microsoft.Azure.Management.MySQL/src/README.md [hunk 2]
```diff
@@ -0,0 +1,2 @@
+# Swagger JSON
+This is a swagger JSON built by the [swagger-codegen](https://github.com/swagger-api/swagger-codegen) project.
```
Comment (allenjzhang) [inline sdk/mysql/Microsoft.Azure.Management.MySQL/src/README.md:2]: nit: Is this file needed. Seems out of context.

**Review by allenjzhang — APPROVED** [review]

File: sdk/mysql/Microsoft.Azure.Management.MySQL/src/Microsoft.Azure.Management.MySQL.csproj (hunk 1 above)
Comment (rajsell) [reply sdk/mysql/Microsoft.Azure.Management.MySQL/src/Microsoft.Azure.Management.MySQL.csproj]: updated
